src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -33,7 +33,7 @@
 
 namespace kafka {
 
-static constexpr std::array<std::string_view, 11> supported_configs{
+static constexpr std::array<std::string_view, 12> supported_configs{
```

Comment:
Nice! 

I was also thinking about here: https://github.com/redpanda-data/redpanda/blob/dev/src/v/kafka/server/handlers/topics/types.cc#L197

These methods wrap and unwrap properties from map<string,string> to well defined types.